design-system/react/src/components/InputAmount/utils.tsx

```diff
@@ -22,3 +23,8 @@ export function createAmount(text: string, units: number = DECIMAL_UNITS) {
     amount: bn.parseUnits(text, units),
   };
 }
+
+export function formatAmount(amount: BNInput, opts: FormatConfig = {}) {
+  const { precision = DECIMAL_UNITS, units = DECIMAL_UNITS } = opts;
```

Comment:
although they are the same number right now, `units` and `precision` are different.

we should use: `DECIMAL_PRECISION` and `DECIMAL_UNITS`